news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – May 2017 edition"
+subtitle: "The latest news from our translation project"
+date:   2017-06-22 06:09:47
+authors: [ alexeven ]
+icon: icon-world
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part to the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! Read this report to see how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2017/04/DYSpeakPS.jpg)
+
+## Project news
+
+### Top languages evolution since January
+
+A lot of words have been translated since the beginning of the year :)
+Here's a quick overview of the evolution of the most active languages over the past few months - Thanks for your involvement!
+
+
+![Crowdin Monthly banner](/assets/images/2017/06/PrestaShop_translation_evolution_2017.gif)
+
+### Deactivating Swiss-German in PrestaShop
+
+It's been a while the German for Switzerland is avalaible for Switzerland merchants. However, this language does not bring any value to merchants at the moment: Indeed, it is similar to the German language, and hasn't been localized to answer the needs of the Swiss market. Confusing.
+Thus, from 1.7.2.O onwards, the Swiss localization package will load the German language only.
+
+The German, Switzerland translation project has also been removed from the Crowdin translation project. It has never really been maintained and it remained inactive for too long.
+
+If in the future, if merchants want to this language back, we'd be happy to re-open the language on Crowdin :)
+
+### Deactivating inactive languages on Crowdin
+
+In the same spirit, other inactive languages have been removed from the Crowdin project:
+
+* Bosnian
+* German, Austria
+* Irish
+* Kannada
+* Malagasy
+* Oriya
+* Spanish, El Salvador
+* Spanish, Honduras
+* Swahili
+* Swahili, Tanzania
+* Telugu
+
+Again, if you wish to contribute for one of these languages, please drop us a word at translation@prestashop.com, and this language will be available for translation again on Crowdin.
+
+### 1.7.2 on its way
+
+As per our release planning, we're preparing version 1.7.2.0.
+
+Before it's release, we will upload its strings to the Crowdin project:
+* so that translators have the time to translate this version before it's release
+* to check the content and see if any modifications are needed before the release
+
+There will be new content, mostly for Stock Management (back in 1.7.2.0), so your help will be required.
```

Comment:
"There will be new content to translate soon, mostly for the Stock Management feature (which is making a come back in 1.7.2.0)" + link to [Guillaume's latest article](http://build.prestashop.com/news/stock-management-in-prestashop-1-7-part-2/)?